layouts/partials/share.html

```diff
@@ -1,8 +1,16 @@
 <span class="flex items-center gap-2">
+  <span class="text-sm font-medium">Share on:</span>
```

Comment:
I'm unwilling to add the text "share on" before icons because extra work is needed to adapt to other languages. And also it's a matter of personal preference, I guess adding a slot there is a better solution so that you can insert anything into it.

Reply:
@diamantelist In [d703a80](https://github.com/g1eny0ung/hugo-theme-dream/commit/d703a803273f94172adf4e0b6206afd173d3797d), I've added a slot named `post-share-left`. Now you can create a `post-share-left.html` file in `layouts/partials/` of your project, and then add this line to it.

I haven't figured out if I need to add the text here, but I think it should also be free for the user to choose. So I added a slot to help implement.